main.m

```diff
@@ -442,13 +442,14 @@ void printKey(NSDictionary *keyItem)
 	printToStdOut(@"Effective Key Size: %@\n", [keyItem objectForKey:(id)kSecAttrEffectiveKeySize]);
 	if ((keySize == effectiveKeySize) && (keySize != 0))
 	{
-		printToStdOut(@"Permanent Key: %@\n", CFBooleanGetValue((CFBooleanRef)[keyItem objectForKey:(id)kSecAttrIsPermanent]) == true ? @"True" : @"False");
```

Comment:
This file has ended up with a funky mix of tabs and spaces. Can you use tabs for now and I'll fix everything up in one commit after this merges?